packages/esbuild/helpers.bzl

```diff
@@ -0,0 +1,127 @@
+"""
+Utility helper functions for the esbuild rule
+"""
+
+# From https://github.com/bazelbuild/bazel-skylib/blob/master/lib/paths.bzl
+def _dirname(p):
```

Comment:
Looks like we already vendor this function in third_party `third_party/github.com/bazelbuild/bazel-skylib/lib/paths.bzl`. You can load it from 
```
load("//third_party/github.com/bazelbuild/bazel-skylib:lib/paths.bzl", "paths")
```
and use it with
```
paths.dirname
```
`internal/node/node_repositories.bzl` has an example of this